### ISSUE TITLE
pysam should be available for python 3.11

### ISSUE THREAD

**Issue by jamestwebber** [bioconda/bioconda-recipes#42796]
The [latest release](https://github.com/pysam-developers/pysam/releases/tag/v0.21.0) of pysam is compatible with python 3.11, but it can't be installed from bioconda. I'm not sure why.

I would open a PR to fix this but I'm genuinely confused where the constraint is coming from. On the [recipe website](https://bioconda.github.io/recipes/pysam/README.html) it says `pysam` has an upper-bound of `python < 3.11.0a0` but I don't see anything about this in the recipe. What is causing that?

**Comment by jmarshall** [comment]
This is not specific to pysam. See #37805.

**Comment by corneliusroemer** [comment]
The answer for now:

>Yes, we intend to support Python 3.11 and 3.12. However, we are currently busy rolling out the ARM builds and in parallel the BioConductor release. This keeps us and the CI already busy. Realistically we can only care about 3.11/12 in January. Sorry.
_(from https://github.com/bioconda/bioconda-utils/pull/938#issuecomment-1846747390)_

**Comment by corneliusroemer** [comment]
Closing as duplicate of #37805
